com.vivaldi.Vivaldi.yaml

```diff
@@ -0,0 +1,124 @@
+app-id: com.vivaldi.Vivaldi
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.chromium.Chromium.BaseApp
+base-version: '22.08'
+command: vivaldi
+separate-locales: false
+finish-args:
+  - --device=all # Webcams and others
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf # For GNOME proxy resolution
+  - --env=GIO_EXTRA_MODULES=/app/lib/gio/modules # For GNOME proxy resolution
+  - --env=GSETTINGS_BACKEND=dconf # For GNOME proxy resolution
+  - --env=GTK_PATH=/app/lib/gtkmodules
+  - --filesystem=/run/.heim_org.h5l.kcm-socket
+  - --filesystem=home/.local/share/applications:create # Allow for PWAs
+  - --filesystem=home/.local/share/icons:create # Allow for PWAs
```

Comment:
This and above should be `xdg-data/icons`, `xdg-data/applications`, I think the current path is just wrong `home/path` isn't valid.

Flatpak has special handling for xdg-dirs, it'll bind mount them

Reply:
I lifted that from Brave if I am honest? Are you saything that one is also wrong?

https://github.com/flathub/com.brave.Browser/blob/master/com.brave.Browser.yaml

Reply:
Can you check if it works? ie  Vivaldi can bot save and load pwas?

Reply:
Also `home/path` is available since flatpak 1.10 while you have `require-version=1.8.2`

Reply:
Ok looks like `home/path` does mostly the same thing:

```
ls -lA ~/.var/app/com.vivaldi.Vivaldi/data/applications
lrwxrwxrwx. 1 wirt wirt 36 Nov  9 21:27 /home/wirt/.var/app/com.vivaldi.Vivaldi/data/applications -> /home/wirt/.local/share/applications
```

Should be fine.